containerized_ml_models/model_api_3/api.py

```diff
@@ -0,0 +1,28 @@
+import json
+
+from flask import Flask, jsonify, request
+import pickle
+import numpy as np
+
+
+app = Flask(__name__)
+
+
+@app.route("/ping", methods=["GET"])
+def ping():
+    return jsonify({"ping": "success"})
+
+
+@app.route("/predict", methods=["POST"])
+def predict():
+    request_data = json.loads(request.data)
+    prediction_data = np.array(request_data)
+
+    model = pickle.load(open("model.pkl", "rb"))

```

Comment:
1. Missed that, thanks
2. While it might be problematic with bigger model (but we're talking of miliseconds, so unnoticable for human API user), I don't see a reason to overengineer it now.